### DISCUSSION TITLE
Script is not running, wondering if this is "Working as Designed" - Maybe CSP Issue?

### DISCUSSION THREAD

**Discussion by thetreythomas** [Tampermonkey/tampermonkey#1721] [Q&A] [answered]
My configuration:
Windows 10 20H2
Chrome Version 110.0.5481.178 (Official Build) (64-bit)
Tampermonkey 4.18.1

I've spent the last hour or so confusing myself even more about Tampermonkey and CSP.

Here is what I am trying to accomplish...

I work with a lot of servers, which have a BMC (Baseboard Management Controller). These BMCs have their own dedicated network port so you can connect to the server even if the server is powered off. If the server is new, or you have cleared the cache on your browser, the first time you go to the BMC address you will always get the message below:

![image](https://user-images.githubusercontent.com/15376624/221794821-d62888c2-5e20-48a8-8b4d-14ab1faddde6.png)

This is going to a URL like `https://192.168.0.1`.

All I was trying to do was write a Tampermonkey script that would click on the "Advanced" button and then click on the "Proceed" link. Figured this would be easy. But nope!

If I manually click on Advanced, and then on the proceed link, I'll land on the BMC homepage. At this point, Tampermonkey scripts work fine.

Do Tampermonkey scripts not run on windows such as these, since it's Chrome stopping before the page loads to display these warnings? I tried all the various settings in the script and in the actual Tampermonkey settings and nothing seems to work.

**Comment by derjanb** [comment, marked as answer]
This is an internal Chrome page that extension can't access.

**Reply by thetreythomas** [reply]
Thanks!
